g-local_vce_detailed.md

```diff
@@ -18,7 +18,7 @@ These instructions describe how to:
 
 - download the module specific Docker container image that contains all the software applications needed for the module
 
-To familiarise yourself with the installation sequence, we recommend that you read through the installation process for your host operating system and the description of testing your installation at least once before carrying out the process on your own computer.
+To familiarise yourself with the installation sequence, we recommend that you read through the installation process for your host operating system and the description of testing your installation at least once before carrying out the process on your own computer. As with all software you only install this software using the verifier installer.
```

Comment:
you >should< only

using the >official< installer from the original publisher's website.

Reply:
Oops that's me missing out words again. And yes your rewording is better.